jetty/src/main/scala/org/http4s/server/jetty/JettyBuilder.scala

```diff
@@ -224,7 +233,8 @@ sealed class JettyBuilder[F[_]] private (
         service = service,
         asyncTimeout = builder.asyncTimeout,
         servletIo = builder.servletIo,
-        serviceErrorHandler = builder.serviceErrorHandler
+        serviceErrorHandler = builder.serviceErrorHandler,
+        dispatcher
```

Comment:
Some other backends are creating `dispatcher` in `Resource`, which is a pattern I like.  The challenge would be how you can thread it down to here.

Reply:
are you suggesting to pass `Resource[IO,Dispatcher[IO]]` instead of `Dispatcher[IO]`

Reply:
No, I'm saying to let one of the functions in this create it.  In BlazeClientBuilder (yes, that's a client, and this is a server, but... almost the same), we're creating and using a `dispatcher` resource in the `.resource` function.  Can we do that here?

I think if the function in the `Mount` class took a `Dispatcher`, the rest of it would fall into place, and you could remove the `Dispatcher` as a parameter of the builder.

Reply:
got it, let me try that

Reply:
@rossabaker the resource function takes returns a resource with type parameter `F`. That means we should make a `Dispatcher[F]` in scope. We can change the signature of `def resource: Resource[F, Server]` to `def resource(dispatcher: Dispatcher[F]): Resource[F, Server]`. But this would mean changing the signature on the trait `ServerBuilder`.
If we want to avoid this, then we would have to pass a dispatcher to the builder.

Reply:
Oh, I was thinking that resource would create a `Dispatcher` resource and flatMap it.  `BlazeClientBuilder` does it like this:

```scala
  def resource: Resource[F, Client[F]] =
    for {
      dispatcher <- Dispatcher[F]
      ...
    } yield ...
```

Reply:
@rossabaker my bad. You are right. Having an implicit `Async[F]` brings `Dispatcher[F]` in scope